### DISCUSSION TITLE
Not installed application

### DISCUSSION THREAD

**Discussion by andrey-helldar** [yhype/yhype-feedback#24] [Bug] [answered]
Hi!

I installed the application in two repositories, but the site has been showing for about an hour that they are not installed. What do I need to do to solve the problem?

<details>
<summary>Screenshots</summary>

![IMG_1041](https://github.com/yhype/yhype-feedback/assets/10347617/502951b7-94d3-4a04-88dc-47f2374868ee)
![IMG_1042](https://github.com/yhype/yhype-feedback/assets/10347617/8fd25dab-cbf6-420e-b824-4de324956198)
![IMG_1044](https://github.com/yhype/yhype-feedback/assets/10347617/999886fb-8163-4ddd-940f-4337a86c482a)

</details>

**Comment by antonkomarev** [comment]
@andrey-helldar I need to check it, will try to make it today.

**Reply by andrey-helldar** [reply]
If it helps, I tried uninstalling the YHYPE app from my personal profile and organization and reinstalling it by clicking on the "Install GitHub App" button on this site, periodically clicking on the "Sync accounts data" button.

All that has changed in the meantime is the "Activity feed" section on the dashboard, but if I go even to my profile, I see the "Install GitHub App" button.

**Reply by antonkomarev** [reply]
@andrey-helldar thanks for the report, it looks like a new bug. Cannot install it on my new organization as well. Going deeper.

**Reply by andrey-helldar** [reply]
If it helps, I used this service a long time ago and decided to return by clicking on the "sign in" button, providing the requested permissions.

What data do I need to provide?

**Reply by antonkomarev** [reply]
@andrey-helldar I'll try to debug it myself, and inform you about the result.

**Reply by antonkomarev** [reply, marked as answer]
@andrey-helldar Try again please, need to uninstall app first. Cool bug. Webhook proxy provider changed an URL >_<

**Reply by antonkomarev** [reply]
Shitty thing that they returned 200 OK to GitHub

**Reply by andrey-helldar** [reply]
Ok. one moment

**Reply by andrey-helldar** [reply]
@antonkomarev, it's works!

<details>
<summary>Screenshots</summary>

![image](https://github.com/yhype/yhype-feedback/assets/10347617/970a02c8-d80f-4cb7-b5f9-8b54f25d1991)
![image](https://github.com/yhype/yhype-feedback/assets/10347617/5cb42191-346c-4f82-abf3-b7e3a50097c9)

</details>

**Reply by andrey-helldar** [reply]
Thank you!

**Reply by antonkomarev** [reply]
@andrey-helldar Thanks a lot! The real help for the service.

**Reply by antonkomarev** [reply]
@andrey-helldar added monitoring for webhooks 🚀

**Reply by andrey-helldar** [reply]
@antonkomarev, another problem related to this one - I can't connect the new organization - https://github.com/cashbox-laravel

<details>
<summary>Screenshots</summary>

![image](https://github.com/yhype/yhype-feedback/assets/10347617/7ea47b59-43ef-4130-9c46-d9a8e25a563c)
![image](https://github.com/yhype/yhype-feedback/assets/10347617/46e639a3-05f0-4923-a3d8-b7d2eca0fd61)
</details>

**Reply by antonkomarev** [reply]
@andrey-helldar Try again please, maybe it happened on metric integration, there was an issue with monitoring code (pushed without tests) :(

**Reply by andrey-helldar** [reply]
@antonkomarev, 
![image](https://github.com/yhype/yhype-feedback/assets/10347617/42dd2667-6796-4514-bb4d-f29bb0017c0f)

Uninstalled the app, reconnected - yes, it appeared. The problem of incorrect repository names remains, which I created another discussion about: https://github.com/orgs/yhype/discussions/26

Thank for fixing!
